dotnet-client-libraries/HandlingDatesWithNodaTime/HandlingDatesWithNodaTime/Extensions/NodaTimeExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NodaTime;
+using NodaTime.Extensions;
+using NodaTime.Text;
+
+namespace HandlingDatesWithNodaTime.Extensions;
+
+public static class NodaTimeExtensions
+{
+    public static Instant ConvertToInstant(this DateTime dateTime)
+    {
+        return dateTime.ToInstant();
+    }
+
+    public static Duration ConvertToDuration(this TimeSpan timeSpan)
+    {
+        return timeSpan.ToDuration();
+    }
+
+    public static LocalDate AddDays(this LocalDate date, int days)
+    {
+        return date.PlusDays(days);
+    }
+
+    public static Period CalculateDifference(this LocalDate startDate, DateTime endDate)
+    {   
+        var finalDate = LocalDate.FromDateTime(endDate);
+        return Period.Between(startDate, finalDate, PeriodUnits.Years);
```

Comment:
Instead of having methods with a single statement just for the sake of the having them in the article, we can create a test case with just the NodaTime method instead of creating a wrapper method like this. 
And in the the article we can have the body of the test in the code snippet for e.g 
the AddDays methods should be a test, something like this : 
```
  [Fact]
  void WhenAddingDaysToDate_ThenHaveCorrectLocalDate() 
   {
        var date = // create LocalDate.
        var threeDaysLater = date.PlusDays(days); // and you can only have this line and the line before in the article and explain it
        // TODO - here assert the resutl
    }
```

Reply:
After you convert those to tests, we can delete this class completely, don't foget to updat the article.